isis/src/base/objs/ImagePolygon/ImagePolygon.cpp

```diff
@@ -852,7 +850,8 @@ namespace Isis {
 
       if (nPoleSample >= 0.5 && nPoleLine >= 0.5 &&
          nPoleSample <= p_cube->sampleCount() + 0.5 &&
-         nPoleLine <= p_cube->lineCount() + 0.5) {
+         nPoleLine <= p_cube->lineCount() + 0.5 &&
+         SetImage(nPoleSample, nPoleLine)) {
```

Comment:
SetImage checks to see if the pixel can get a valid projection when going image->ground. 

idk why ground->image successfully returns pixel coords but image->ground acknowledges that there is no intersection?